src/cascadia/WindowsTerminal/IslandWindow.h

```diff
@@ -74,6 +76,9 @@ class IslandWindow :
 
     LONG _getDesiredWindowStyle() const;
 
+    ITaskbarList3* _taskbar;

```

Comment:
raw pointers are scary - can we wrap this in a `ComPtr` or something? (/cc @miniksa who could actually help with the right syntax here)

Reply:
I'd suggest adding some comments about the ownership of this pointer to make it less scary.

Reply:
`wil::com_ptr<ITaskbarList3>` should do the needful here.